h2o-core/src/main/java/hex/Model.java

```diff
@@ -257,6 +257,9 @@ public void setTrain(Key<Frame> train) {
       final boolean _needResponse;
       boolean needsResponse() { return _needResponse; }
     }
+
+    public Key<Model> _te_model;
```

Comment:
In the middle of nowhere. Let's move it up a little bit or group it with other TE methods.